protobuf/policy.proto

```diff
@@ -14,8 +14,17 @@ message HealthCheckReply {
   int32 Retval = 1;
 }
 
+enum PolicyStatus {
+  Failure = 0 ;
+  Applied = 1 ;
+  Deleted = 2 ;
+  Modified = 3 ;
+  NotExist = 4;
+  Invalid = 5;  
```

Comment:
Can we use negative values for these?

Reply:
Using negative values is not recommended in grpc documentation. [Ref](https://protobuf.dev/programming-guides/proto3/#:~:text=Since%20enum%20values%20use%20varint,inefficient%20and%20thus%20not%20recommended.)